dojo/tools/chefinspect/parser.py

```diff
@@ -0,0 +1,52 @@
+import json
+from dojo.models import Finding
+
+
+class ChefInspectParser(object):
+    def get_scan_types(self):
+        return ["Chef Inspect Log"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return """Chef Inspect log file"""
+
+    def convert_score(self, raw_value):
```

Comment:
It appears [from their documentation](https://docs.chef.io/inspec/profiles/controls/#control-properties) that `impact` of [0.0,1.0] is interpreted like a score from 0-100, with the following severity breakdowns.

![Screenshot 2024-01-16 at 13 12 51](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/7cfb2079-5c4c-4717-b1b1-5ee7bac1a5dd)

You can see this in practice in e.g. the [`dev-sec` MySQL baseline](https://dev-sec.io/baselines/mysql/)

![Screenshot 2024-01-16 at 13 09 18](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/de3a0a94-5525-4b25-9463-d26e99629be5)

Reply:
@manuel-sommer I approved too quickly.. the data in the sample files provided also do not have an impact greater than 1.0, so I think the above may be the source to go with